contracts/vaults/NonvotingVault.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.3;
+
+import "../libraries/Authorizable.sol";
+import "../interfaces/ILockingVault.sol";
+import "../interfaces/IERC20.sol";
+
+contract NonvotingVault is Authorizable {
+    // The locking gov vault
+    ILockingVault public lockingVault;
+    IERC20 public immutable token;
+
+    /// @notice Constructs this contract
+    /// @param _owner The address authorized to withdraw
+    /// @param _lockingVault The governance vault which this withdraws from
+    constructor(address _owner, ILockingVault _lockingVault) Authorizable() {
```

Comment:
FYI - Just want to mention you don't need to explicitly mention the constructor of the inherited contract if it isn't taking the parameters.

Reply:
I think it's actually good for code readability to declare the constructor of the inherited contract. That way there's no ambiguity about what code is running on deployment.

Reply:
Agree with Paul, gonna keep it in the constructor